src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.h

```diff
@@ -7,16 +7,19 @@
 
 namespace kernel_selector {
 
+enum class roi_aligned_mode : uint32_t { asymmetric, half_pixel_for_nn, half_pixel };
```

Comment:
haven't we use caps for enum values in these files?

Reply:
Done